sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -57,8 +63,13 @@ export class BatchingReceiver extends EventHubReceiver {
       let onSessionClose: OnAmqpEvent;
       let waitTimer: any;
       let actionAfterWaitTimeout: Func<void, void>;
+      let aborter: Aborter;
+      let onAbort: () => {};
```

Comment:
The parent class has the both these properties defined on it. Why not use them instead of re-defining them here?